docs/reference/remap/functions/parse_aws_cloudwatch_log_subscription_message.cue

```diff
@@ -36,23 +36,23 @@ remap: functions: parse_aws_cloudwatch_log_subscription_message: {
 					{
 					  "id": "35683658089614582423604394983260738922885519999578275840",
 					  "timestamp": 1600110569039,
-					  "message": "{\"bytes\":26780,\"datetime\":\"14/Sep/2020:11:45:41 -0400\",\"host\":\"157.130.216.193\",\"method\":\"PUT\",\"protocol\":\"HTTP/1.0\",\"referer\":\"https://www.principalcross-platform.io/markets/ubiquitous\",\"request\":\"/expedite/convergence\",\"source_type\":\"stdin\",\"status\":301,\"user-identifier\":\"-\"}"
+					  "message": "{\\"bytes\\":26780,\\"datetime\\":\\"14/Sep/2020:11:45:41 -0400\\",\\"host\\":\\"157.130.216.193\\",\\"method\\":\\"PUT\\",\\"protocol\\":\\"HTTP/1.0\\",\\"referer\\":\\"https://www.principalcross-platform.io/markets/ubiquitous\\",\\"request\\":\\"/expedite/convergence\\",\\"source_type\\":\\"stdin\\",\\"status\\":301,\\"user-identifier\\":\\"-\\"}"
```

Comment:
This would probably be cleaner to enclose the outer string in `'''` instead of double escaping the slashes - at least I think that would work in cue. But it's not the end of the world like this..